### DISCUSSION TITLE
How do I add specific tags to Regex notes?

### DISCUSSION THREAD

**Discussion by HeroJeanPierre** [ObsidianToAnki/Obsidian_to_Anki#370] [Q&A]
I am using [Q & A Style](https://github.com/Pseudonium/Obsidian_to_Anki/wiki/Question-answer-style) and unsure how to add tags. I am using the suggested format posted in the [regex](https://github.com/Pseudonium/Obsidian_to_Anki/wiki/Regex) page but it's not working, here is an example card.

Q: How far is the earth from the moon?
A: 238,900 Miles #science
Tags: science

This simple adds the string "Tags: science" to the back of the card. Unsure what to do.

Edit: I even attempted forking the files mentioned here, #230 , no luck.

**Comment by HeroJeanPierre** [comment]
Found solution here: https://github.com/Pseudonium/Obsidian_to_Anki/issues/196#issuecomment-762487077
Note, I forked the repo at #230 already, if it doesn't work try forking the repo as explained in that thread, though it may work without forking.

**Reply by quake1508** [reply]
This only works when using text.
When using **mathmode** for latex: $$ for example be sure to write some text afterwards. Otherwise your tags won't be seen as tags but as text within the card.

e.g.:
```{
Q: Wie lauten die Londongleichungen? 

A: test 
$$
\dot{\vec{j}}=\frac{n_{s}e^{2}}{m}\vec{E}
$$
und die 2. $$
\mathrm{rot}\vec{j}=-\frac{n_{s}e^{2}}{m}\vec{B}
$$ags: London-Gleichungen

```

Gives the following result:
![image](https://github.com/user-attachments/assets/f1a4dc72-ecc6-4606-b629-26658f980785)

Whereas:
```{
Q: Wie lauten die Londongleichungen? 

A: test 
$$
\dot{\vec{j}}=\frac{n_{s}e^{2}}{m}\vec{E}
$$
und die 2. $$
\mathrm{rot}\vec{j}=-\frac{n_{s}e^{2}}{m}\vec{B}
$$Tagszeile Tags: London-Gleichungen
```

gives:
![image](https://github.com/user-attachments/assets/6d2776fb-d9ff-45ad-ab68-d5d40801ea23)

I don't know whether this is a bug or wanted. I might submit an issue, although i can live with it for now

Beware: this workaround only works when creating the new ankicard.
When editing an old ankicard through Obsidian you won't be able to edit the tags by simply doing the workaround i mentioned.
